### DISCUSSION TITLE
OTLP to Zipkin BE for Trace exporting

### DISCUSSION THREAD

**Discussion by lxx95** [open-telemetry/opentelemetry-demo#881] [Q&A] [answered]
Hi all,

Trying to export traces from Otel-Demo (with receiver in OTLP format) to Zipkin backend.

Here my otel-collector configmap configuration:

![image](https://user-images.githubusercontent.com/57501699/236671193-bf7cf461-4018-4ef1-b429-e86463d4ac15.png)

This error message has been shown for exporting phase (from otel-collector):
![image](https://user-images.githubusercontent.com/57501699/236671231-47eaccd3-066b-474b-b9f6-5e444226e1e2.png)

The zipkin backend port and endpoint was fine:
![image](https://user-images.githubusercontent.com/57501699/236671274-bb92c205-1767-41bb-8e07-9f5d4fd95a9d.png)
![image](https://user-images.githubusercontent.com/57501699/236671348-7dab1743-c22c-4a04-9260-b7168d23e71d.png)

I have opened also [another discussion](https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/21492) on opentelemetry-collector-contrib repo for my final purpose (use another backend, Skywalking, that is compatible with zipkin protocol).
The same result was shown so, I think that the problem is about collector/otel-demo compatibility.

Thank you,
Luigi

**Comment by puckpuck** [comment, marked as answer]
the error seems to indicate that the zipkin exporter on the collector needs an `http://` or `https://` prefix (scheme) on your endpoint.
